### PR TITLE
Add details on the namespaces in the FAQ

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Added FAQ details on the namespaces

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: docs/data-warehouse-integrations/warehouse-faqs.mdx [hunk 1]
```diff
@@ -152,6 +143,30 @@ To update the namespace as the new source name, you need to configure a new sour
...
+### How is the schema name configured for my data warehouse?
+
+By default, the schema name is set as:
+- The Namespace field value in the RudderStack dashboard. **OR**
+- The Source name value in the RudderStack dashboard, if the above is not provided.
+
+However, the following scenarios with examples provide more details on how the schema name is configured based on the presence/absence of Namespace field:
+
+**When Namespace field is present**
+- If the Namespace field is set as "ABC" and the Source name is "X", the schema name would be "ABC". However, if you change the Namespace field to "DEF" later on, the schema name would become "DEF".
```
Comment (ameypv-rudder) [inline docs/data-warehouse-integrations/warehouse-faqs.mdx:164]: Would it be better to present this information as @Jayachand suggested here: https://rudderlabs.slack.com/archives/C02AALVFHJN/p1654146806831279?thread_ts=1654048885.999709&cid=C02AALVFHJN

Comment (ishachopra0401) [reply docs/data-warehouse-integrations/warehouse-faqs.mdx:164]: Yup, sure!

**Review by ameypv-rudder — APPROVED** [review]

**Review by achettyiitr — APPROVED** [review]
LGTM

**Review by Jayachand — APPROVED** [review]

**Comment by ameypv-rudder** [comment]
@ishachopra0401 as discussed - we can copy and paste the 2 FAQs in the main branch and close this PR.

**Comment by ameypv-rudder** [comment]
The changes have been incorporated in `main`: [reference](https://github.com/rudderlabs/rudderstack-docs/commit/34153c0ffba2884bd4f415f0773d062096ef4c27)

**Pull request closed without merging**